source/Clean.js

```diff
@@ -179,7 +179,9 @@ var stylesRewriters = {
     }
 };
 
-var allowedBlock = /^(?:A(?:DDRESS|RTICLE|SIDE|UDIO)|BLOCKQUOTE|CAPTION|D(?:[DLT]|IV)|F(?:IGURE|IGCAPTION|OOTER)|H[1-6]|HEADER|L(?:ABEL|EGEND|I)|O(?:L|UTPUT)|P(?:RE)?|SECTION|T(?:ABLE|BODY|D|FOOT|H|HEAD|R)|COL(?:GROUP)?|UL)$/;
```

Comment:
I don't see why you are wrapping this in an object.

Reply:
I want to overwrite `inlineNames` and `allowBlock`. Because `config` should be merged only with object, I wrapped `inlineNames` and `allowBlock`.

https://github.com/neilj/Squire/blob/892986b17c60b9437abd99f8a7ed5d24d8e52294/source/Editor.js#L164

Please let me know if there are any other ways I can add those as a config.